xml/security_cryptofs.xml

```diff
@@ -269,4 +278,83 @@
     the file.
   </para>
  </sect1>
+ <sect1 xml:id="sec-security-cryptofs-rage">
+  <title>Encrypting files with Rage</title>
+  <para>
+   Rage is a secure file encryption software to encrypt files.
+   Rage has small explicit keys and does not have configuration options.
+  </para>
+  <para>You must first generate a key pair to encrypt a file with Rage:
+   </para>
+   <screen>&prompt.user;rage-keygen -o ~/rage.key 2 ~/rage.pub</screen>
+   <para>Note that two files are created; <emphasis>rage.pub</emphasis> and <emphasis>rage.key</emphasis>.
+   A recipient can be either an <emphasis>age</emphasis> or <emphasis>SSH</emphasis> public key.</para>
+   <para><emphasis role="bold">rage.pub example</emphasis></para>
+   <screen>&prompt.user; cat file.pub
+    Public key: age17e4g67cs07jk3lmylyq6gduv26uf7tz7nm9jrsaxn8xxx9uc9amsdg4a5e </screen>
+    <para><emphasis role="bold">rage.key example</emphasis></para>
+   <screen>&prompt.user; cat file.key
+    # created: 2023-05-30T16:29:20+05:30
+    # public key: age17e4g67cs07jk3lmylyq6gduv26uf7tz7nm9jrsaxn8xxx9uc9amsdg4a5e </screen>
+   <para><emphasis role="bold">Encrypt</emphasis></para>
+   <para>To encrypt a file, you need the generated public key:</para>
+   <screen>&prompt.user;rage -e -r <replaceable>PUBLIC_KEY</replaceable> -o <replaceable>ENCRYPTED_FILE</replaceable> <replaceable>FILE</replaceable> </screen>
+   <para>For example:</para>
+   <screen>>&prompt.user;rage -e -r age17e4g67cs07jk3lmylyq6gduv26uf7tz7nm9jrsaxn8xxx9uc9amsdg4a5e -o test.txt.age test.txte </screen>
+   <para>An encrypted version of the specified file is recognizable by the <emphasis>.age</emphasis> file extension that is created.</para>
```

Comment:
This is false, you name the encrypted output whatever the user wants with an extension they want.